### PR TITLE
Remove "caveat" sections from top of page, create new footer

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: webapp/static/common.css [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
+  font-size: 1.5em;
+  margin-bottom: 0.5em;
 }
 article p, ul {
-    margin-bottom: 1em;
+  margin-bottom: 1em;
 }
 article ul {
-    margin-left: 2em;
-}
```
Comment (rwaldron) [inline webapp/static/common.css]: All of these changes are whitespace/indentation.

File: webapp/templates/_header.html [hunk 2]
```diff
@@ -7,4 +7,7 @@ <h1><a href="/">web-platform-tests dashboard</a></h1>
     <a href="/about">About</a>
     <a href="https://github.com/w3c/wptdashboard">GitHub Source</a>
   </nav>
+  <section class="caveat">
+    This site is a work-in-progress.
```
Comment (rwaldron) [inline webapp/templates/_header.html]: @jugglinmike please advise here

File: webapp/static/common.css (hunk 1 above)
Comment (jugglinmike) [reply webapp/static/common.css]: In the future, a dedicated commit would be nice (as a GitHub.com comment your explanation won't show up in the git history). I don't think it's worth the effort to change it here, though

File: webapp/templates/_footer.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<footer>
+  <section class="caveat">
+    This information is intended for web platform implementers and does not contain
+    useful metrics for evaluation or comparison of web platform features.
```
Comment (jugglinmike) [inline webapp/templates/_footer.html]: How about:

"The Web Platform Tests Dashboard is a work-in-progress. The reported results do not necessarily reflect the true capabilities of each web browser."

File: webapp/static/common.css [hunk 4]
```diff
@@ -1,51 +1,58 @@
...
\ No newline at end of file
+  margin-left: 2em;
+}
+.caveat {
+  margin-top: 20px;
+  padding: 0.5em;
```
Comment (jugglinmike) [inline webapp/static/common.css]: Could we stick with `em` units for consistency's sake?

File: webapp/templates/_footer.html (hunk 3 above)
Comment (jugglinmike) [reply webapp/templates/_footer.html]: Oh, err, I think maybe the text I'm recommending belongs in the header.

File: webapp/static/common.css (hunk 1 above)
Comment (rwaldron) [reply webapp/static/common.css]: Noted!

File: webapp/static/common.css (hunk 4 above)
Comment (rwaldron) [reply webapp/static/common.css]: lol, sorry—that was lazy ;)

File: webapp/templates/_footer.html (hunk 3 above)
Comment (rwaldron) [reply webapp/templates/_footer.html]: I've updated the top banner :)

**Comment by jugglinmike** [comment]
Now that I've finally run this locally and seen the visual design, I have some additional feedback on the copy itself. It's a lot of text, and its distinctive visual treatment is a bit jarring when presented across three elements.

Simply combining the messages is probably not a good idea, since that would result in a lengthy paragraph above-the-fold. That paragraph also wouldn't read so well because each sentence was written independently.

What do you think about re-writing the copy to be more terse and including that shorter text in the header and footer of every page?

More concretely, I'm thinking we can go from this:

> > The Web Platform Tests Dashboard is a work-in-progress. The reported results do not necessarily reflect the true capabilities of each web browser.
>
> [...]
>
> > Data below are intended for web platform implementers and do not contain useful metrics for evaluation or comparison of web platform features. Each browser listed here is currently tested at its latest stable release. See #109 and #112 for discussion on testing pre-release versions. 
>
> [...]
>
> > This information is intended for web platform implementers and does not contain useful metrics for evaluation or comparison of web platform features. 

To something like this:

> The Web Platform Tests Dashboard is a work-in-progress. The reported results do not necessarily reflect the true capabilities of each web browser, so they should not be used evaluate or compare feature support. Each browser listed here is currently tested at its latest stable release.

(Apologies for the delay on this feedback--I spent the morning struggling with the proprietary Google technologies that the project depends on.)

**Comment by rwaldron** [comment]
@jugglinmike updates made in new commit

**Comment by rwaldron** [comment]
Merged

**Comment by jugglinmike** [comment]
Sorry, I'm still new to the project. How do these changes get deployed?

**Comment by rwaldron** [comment]
@jugglinmike I can understand how that might be frustrating, considering there is almost no documentation available in this repository.

**Pull request closed without merging**